tests/python/test_api.py

```diff
@@ -44,7 +44,34 @@ def _get_expected_matrix_apis():
         'normalized',
         'one',
         'outer_product',
+        'translate',
+        'scale',
+        'eulerAngleX',
+        'eulerAngleY',
+        'eulerAngleZ',
+        'eulerAngleXY',
+        'eulerAngleYX',
+        'eulerAngleXZ',
+        'eulerAngleZX',
+        'eulerAngleYZ',
+        'eulerAngleZY',
+        'eulerAngleXYZ',
+        'eulerAngleYXZ',
+        'eulerAngleXZX',
+        'eulerAngleXYX',
+        'eulerAngleYXY',
+        'eulerAngleYZY',
+        'eulerAngleZYZ',
+        'eulerAngleZXZ',
+        'eulerAngleXZY',
+        'eulerAngleYZX',
+        'eulerAngleZYX',
+        'eulerAngleZXY',
+        'yawPitchRoll',
+        'rotate_by_vector',
```

Comment:
The function naming convention is not consistent.

Reply:
I'm not not quite sure what you mean

Reply:
There are `snake_case` and `camelCase` in the list

Reply:
sorry, I will convert all `camelCase ` to `snake_case `